### DISCUSSION TITLE
Stuck on Caddy for Let’s Encrypt method

### DISCUSSION THREAD

**Discussion by swetepete** [coder/code-server#5069] [Q&A]
I’d like to access my code-server from a Firefox browser on a computer where I don’t have terminal access (library computer).

I was planning on doing the Let’s Encrypt installation route.

I am stuck on installing Caddy. The suggested script says that Java isn’t installed, even though I installed it with Homebrew. You can also Homebrew install Caddy, but the next step is to edit the /etc/caddy file, which hasn’t appeared after the brew installation.

Would anyone have any advice?

Is an SSH tunnel from Firefox possible?

Why isn’t the file there after the brew install?

Why does the script think Java isn’t installed?

Thanks very much

**Comment by swetepete** [comment]
Same with nginx, I can’t find the sites-available file after brew installation.

**Comment by jsjoeio** [comment]
Hmm...thinking about this out loud. So is code-server installed on the library computer, or are you simply accessing it via the library computer?

One quick option (without worrying about Caddy/nginx) is to use something like `ngrok` or `localtunnel`. I guess you'd have to:
1. SSH into your machine where code-server is
2. use something like `localtunnel`
3. at the library computer, access code-server

Hmm..can you post the logs from installing Caddy? Let's start there
